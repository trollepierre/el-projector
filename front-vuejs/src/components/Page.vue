<template>
  <div class="page">
    <TaskTable />
  </div>
</template>

<script>
import TaskTable from './TaskTable.vue'

export default {
  name: 'HelloWorld',
  components: { TaskTable },
}
</script>

<style scoped>
h1 {
  font-weight: normal;
}
</style>
