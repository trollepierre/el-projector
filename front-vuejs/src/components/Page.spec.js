import Page from './Page.vue'

describe('Page.vue', () => {
  it('should match snapshot', () => {
    // When
    const wrapper = shallowMount(Page)

    // Then
    expect(wrapper.element).toMatchSnapshot()
  })
})
