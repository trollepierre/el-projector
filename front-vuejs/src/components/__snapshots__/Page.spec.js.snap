// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page.vue should match snapshot 1`] = `
<div
  class="page"
>
  <taskstable-stub />
</div>
`;
